tests/test_pyproject_pep517.py

```diff
@@ -69,6 +70,8 @@ def test_pep517_sdist(tmp_path, monkeypatch):
 
 @mark_hashes_different
 def test_pep517_sdist_hash(tmp_path, monkeypatch):
+    # Unset SOURCE_DATE_EPOCH in order to guarantee the hash match
+    os.environ.pop("SOURCE_DATE_EPOCH", None)
```

Comment:
```suggestion
    monkeypatch.delenv("SOURCE_DATE_EPOCH", raising=False)
```

Reply:
The monkeypatch will restore the environment variable after the test finishes, so you won't get order of tests creating different environments.